neural_compressor/adaptor/torch_utils/gptq.py

```diff
@@ -831,6 +831,9 @@ def fasterquant(self, W, blocksize=128, percdamp=0.01, groupsize=-1, act_order=F
                         if act_order:
                             idx = perm[idx]
                         self.quantizer = groups[idx // groupsize]
+                        if (i1 + i) % groupsize == 0:

```

Comment:
do we have UT to cover?

Reply:
Of course. I have solved the exporting problem, and I will add UTs a.s.a.p.